src/debuggability.js

```diff
@@ -141,6 +141,69 @@ Promise.hasLongStackTraces = function () {
     return config.longStackTraces && longStackTracesIsSupported();
 };
 
+var supportedEvents = ["created", "chained", "fulfilled",
+    "rejected", "resolved", "cancelled"];
+
+Promise.on = function (eventName, hookFunction, simpleEventApi) {
+    if (supportedEvents.indexOf(eventName)<0) {
```

Comment:
Please use white space  consistent  with the project:

`if (supportedEvents.indexOf(eventName) < 0) {`